src/theme/theme.js

```diff
@@ -2,140 +2,129 @@ import { createTheme } from "@mui/material/styles";
 
 const theme = createTheme({
   palette: {
-    type: "dark",
+    type: "light",
```

Comment:
@Poonam-Singh-Bagh From https://mui.com/guides/migration-v4/, <q>The `theme.palette.type` key was renamed to `theme.palette.mode`, to better follow the "dark mode" term that is usually used for describing this feature.</q> 
```suggestion
    mode: "light",
```

Reply:
Sure. Thanks.